recipes/simpervisor/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "simpervisor" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d82e4527ae326747551e4bdfa632ff4ebef275ce721f80886c747adfdbf41c2e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+
+test:
+  imports:
+    - simpervisor
+
+about:
+  home: https://github.com/yuvipanda/simpervisor
+  license: BSD-3-Clause
+  # Vendoring the license file until the packages include it.
+  # xref: https://github.com/yuvipanda/simpervisor/pull/5
+  license_file: LICENSE
```

Comment:
Submitted PR ( https://github.com/yuvipanda/simpervisor/pull/5 ) to package the license file in PyPI packages. Have vendored it in the recipe for now.